### ISSUE TITLE
Playerversion Handling: Zugriff auf ältere Playerversion bei mehreren Playern

### ISSUE THREAD

**Issue by tobhu1981** [iqb-berlin/testcenter-frontend#403]
Ich räume gerade alte Trello Boards auf. Dabei ist mir folgende ältere Karte aufgefallen:

JK:
Wir haben für die Abgabe Aufgaben mit verschiedenen Playerversionen (v1.4 und v1.5) ins Testcenter geladen. Beide Player liegen ebenfalls in dem Arbeitsbereich. Wenn man sich dann mit einem Login im Reviewmodus den Test ansieht, ist es so, dass von allen Aufgaben, unabhängig davon welche Playerversion sie eigentlich nehmen sollten, die alte Version 1.4 genommen wird.

So was ähnliches hatte mir BW auch schon einmal gemeldet. Ihr habt das glaube ich laut Martin auch auf dem Schirm und es sollen da wohl dsbzgl. Änderungen im TC folgen. Ich weiß nur nicht wie der Stand ist und wollte nicht dass die Trellokarte ganz verschwindet, daher jetzt hier das Ticket.

**Comment by paflov** [comment]
Die Player-Auswahl-Logik folgt zwei Regeln:

1. Gibt es exakt die Version (Dateiname), die in der Unit definiert, wird diese Verwendet. 

2. Gibt es ihn nicht, wird die neuste passende verwendet.

Beispiel: (links die verfügbaren Dateien, oben das gesetzte Attribut in der Unit.xml, in der Tabelle was gewählt werden würde. Fett, wenn  Regel 1 greift.)
|                                                         	| xx-1.html | xx-1.2.html 	| xx-1.2.0.html 	| xx-1.2.1.html 	| xx-1.2.3.html 	|
|---------------------------------------------------------	|--------------------	|----------------------	|------------------------	|------------------------	|------------------------	|
| xx-1.2.0.html xx-1.2.3.html xx-1.2.4.html xx-1.3.0.html 	| xx-1.3.0.html      	| xx-1.2.4.html        	| **xx-1.2.0.html**      	| xx-1.2.4.html          	| **xx-1.2.3.html**      	|
| xx-1.2.html xx-1.2.3.html xx-1.2.4.html xx-1.3.0.html   	| xx-1.3.0.html      	| **xx-1.2.html**      	| xx-1.2.4.html          	| xx-1.2.4.html          	| **xx-1.2.3.html**      	|

Die Änderungen, die geplant sind sind folgende: Die Player-Version wird anhand ihrer Metadaten erkannt, nicht anhand ihres Dateinamens, die Logik bleibt jedoch die selbe.

**Comment by tobhu1981** [comment]
Okay....danke dir für die Erklärung. Soll ich das mal noch an geeigneter Stelle ins Wiki schreiben?

**Comment by paflov** [comment]
Ja, würde ich schon sagen. Wer weiß, wann wir dazu kommen das zu ändern. Bis dahin sollte es irgendwo stehen, zumal das Thema in letzter Zeit öfter kam.

Das Ticket kann dann zu, oder?

**Comment by tobhu1981** [comment]
Weiß nicht...zu machen würde ich es vielleicht noch nicht. Vielleicht lieber auf hold-on oder icebox oder so???

**Comment by jkoelm** [comment]
Philipp, ich weiß nicht, ob das stimmt. Bei uns war den Aufgaben eine eindeutige Version zugeschrieben, aber es würde trotzdem überall die ältere Playerversion genommen (siehe Eintrag von Tobias ganz oben).

**Comment by paflov** [comment]
Lieber Tobias,
kannst du bei Gelegenheiten noch mal genau testen ob es hier ein Fehlverhalten gibt und dokumentieren bie welchem Fall genau? Das gewünschte Verwalten siehe oben.
Falls alles reibungslos funktioniert, kannst du das Ticket gerne Schließen.
